src/components/containers/AdminDashboard/AdminWorkshopsContainer.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import WorkshopsOverview from "../../presentational/AdminDashboard/WorkshopsOverview";
+import CenteredTabs from "../../presentational/CenteredTabs";
+import { Typography } from "@material-ui/core";
+import { RouteComponentProps } from "react-router";
+
+type AdminWorkshopsContainerProps = RouteComponentProps<any> & {};
+
+const workshopTabs = [
+	{
+		title: "Current"
+	},
+	{
+		title: "Archived"
+	}
+];
+
+const AdminWorkshopsContainer: React.SFC<AdminWorkshopsContainerProps> = props => {
+	const { path } = props.match;
+	const [state, setState] = React.useState({
+		tab: "Current"
+	});
+
+	const myCallbackTab = (tabSelected: string) => {
+		setState({
+			tab: tabSelected
+		});
+	};
+
+	return (
+		<div>
+			<br />
+			<Typography variant="h3">Workshops</Typography>
+			<CenteredTabs workshopTabs={workshopTabs} props={props} callbackTab={myCallbackTab} />
```

Comment:
I don't know what the current UI is but i'm not sure if having an additional navbar is a good idea. If we just need 2 buttons on the page, just create a div with the two buttons inside. If this is an agreed upon design and is what we want to do though then sure we can keep it.